packages/@okta/vuepress-site/docs/reference/api/asa/projects/index.md

```diff
@@ -0,0 +1,2012 @@
+    "created_by_user": "william.faulkner",
+    "issued_at": "2020-10-02T06:54:46.910552Z",
+    "description": "Test Server Enrollment Token"
+}
+```
+### Delete a Server Enrollment Token from a Project
+
+<ApiOperation method="DELETE" url="https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/server_enrollment_tokens/${server_enrollment_token_id}" />
+This endpoint deletes a Server Enrollment Tokens from a Project.
+
+This endpoint requires one of the following roles: `access_admin`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `project_name`   | string | The Project name. |
+| `server_enrollment_token_id`   | string | The UUID of the Server Enrollment Token. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns a `204 No Content` response on a successful call.
+
+
+#### Usage example
+
+##### Request
+
+```bash
+curl -v -X DELETE \
+-H "Authorization: Bearer ${jwt}" \
+https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/server_enrollment_tokens/${server_enrollment_token_id}
+```
+
+##### Response
+
+```json
+HTTP 204 No Content
+```
+### List Server Users in a Project
+
+<ApiOperation method="GET" url="https://app.scaleft.com/v1/teams/${team_name}/projects/${project_name}/server_users" />
+List all the Server Users in a Project. A Server User is a representation of a given ASA User that that will be created on an end server.
+
+This endpoint requires one of the following roles: `access_user`, `access_admin`, `reporting_user`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `project_name`   | string | The Project name. |
+| `team_name`   | string | The name of your Team. |
+
+
+#### Request query parameters
+
+This endpoint has no query parameters.
+
+#### Request body
+
+This endpoint has no request body.
+
+#### Response body
+This endpoint returns a list of objects with the following fields and a `200` code on a successful call.
+| Properties | Type        | Description          |
+|----------|-------------|----------------------|
+| `admin`   | boolean | True if Server User has sudo permissions. |
+| `id`   | string | UUID of Server User API object. |
+| `server_user_name`   | string | The username that will be realized on Unix servers. |
+| `status`   | string | Status of the Server User. |
+| `type`   | string | Whether this is a Service or human user. |
+| `unix_gid`   | number | Unix GID of the Server User. |
+| `unix_uid`   | number | Unix UID of the Server User. |
+| `user_name`   | string | The username of the corresponding ASA User. |
+| `windows_server_user_name`   | string | The username that will be realized on Windows servers. |
```

Comment:
```suggestion
| `windows_server_user_name`   | string | The username that is used on Windows servers. |
```
Please do this for all `windows_server_user_name` table entries in this section.